package.json

```diff
@@ -57,7 +57,7 @@
     "classnames": "2.2.6",
     "common-tags": "1.8.0",
     "dom-helpers": "3.3.1",
-    "downshift": "2.2.3",
+    "downshift": "3.1.1",
```

Comment:
Should do this in the main repo too. Are you aware of any impactful changes on us there?

Reply:
ui-kit barely uses any features of downshift, so the breaking changes didn't affect us. I'd recommend to read through the changelog for the main repo as some things were renamed.

Reply:
ui-kit barely uses any features of downshift, so the breaking changes didn't affect us. I'd recommend to read through the changelog for the main repo as some things were renamed.

Reply:
ui-kit barely uses any features of downshift, so the breaking changes didn't affect us. I'd recommend to read through the changelog for the main repo as some things were renamed.

Reply:
ui-kit barely uses any features of downshift, so the breaking changes didn't affect us. I'd recommend to read through the changelog for the main repo as some things were renamed.